### PR TITLE
UD-1022 add list of user to admin management section

### PR DESCRIPTION
**Pull request by olexii4**
@ashumilova @benoitf  WDYT?

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "angular-gravatar": "~0.2.4",
     "angular-material": "~1.0.1",
     "angular-moment": "~0.9.0",
-    "angular-messages": "~1.4.8",
```
Comment (benoitf) [inline bower.json]: why the version is being downgraded ?

File: bower.json [hunk 2]
```diff
@@ -30,7 +30,7 @@
     "jsonlint": "~1.6.0",
     "ng-clip": "~0.2.6",
     "ng-lodash": "~0.2.3",
-    "ng-password-strength": "~0.1.0",
+    "ng-password-strength": "~0.2.1",
```
Comment (benoitf) [inline bower.json]: why version is updated ?

File: src/app/admin/user-management/user-management.styl [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+ * Copyright (c) 2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ */
+$title-color = #dfdfdf
```
Comment (benoitf) [inline src/app/admin/user-management/user-management.styl]: this color definition should not be in a child stylus file, it's harder to track

File: bower.json (hunk 1 above)
Comment (olexii4) [inline bower.json]: Sorry. It is an old version(merge mistake) . It should be - 1.4.8

File: bower.json (hunk 2 above)
Comment (olexii4) [inline bower.json]: Becouse in  version  > 0.2.0 dependency on bootstrap was removed. And it is fine.

Comment (benoitf) [inline bower.json]: ok maybe a separate commit then

File: bower.json (hunk 1 above)
Comment (benoitf) [inline bower.json]: ok

File: src/app/admin/user-management/user-management.styl (hunk 3 above)
Comment (benoitf) [inline src/app/admin/user-management/user-management.styl]: maybe we can reuse existing color definition among
  "$cat-gray-color": "#E4E4E4",
  "$mouse-gray-color": "#D3D3D3",
  "$light-gray-color": "#F1F1F1",

**Comment by ashumilova** [comment]
+1

**Pull request merged**